### PR TITLE
Add new sections about types to the tutorial

### PR DESCRIPTION
**Pull request by david-a-wheeler**

### CODE REVIEW

**Comment by david-a-wheeler** [comment]
@expln - I added this text as inspired by your video:

https://drive.google.com/file/d/13ihXqjKTab1RAJsr-V70_Pl4oFukdKAl/view?usp=sharing

If I got something *wrong* please let me know. I decided to not focus on the "why some things get selected", I think that's too complicated at this point in the material.

**Comment by expln** [comment]
Hi David. I will review this PR in approx. 12 hrs. Please feel free to modify it if needed.

**Review by expln — APPROVED** [review]
> If I got something wrong please let me know.

Everything is correct. Thank you!

> I decided to not focus on the "why some things get selected", I think that's too complicated at this point in the material.

I agree. The guide already contains a lot of useful information. It is more than enough for the beginning :)

File: docs/11/index.md [hunk 1]
```diff
@@ -966,44 +967,203 @@ See [common set.mm symbols](#common-setmm-symbols) for more.
...
+In this step, we substitute `ps` and `ch`, producing `( ps -> ch )`.
+
+This reveals how Metamath ensures every expression has the correct type
+in an existing proof.
+
+#### Revealing more about Metamath types in the editor
+
+<!-- https://drive.google.com/file/d/13ihXqjKTab1RAJsr-V70_Pl4oFukdKAl/view -->
+You can also make the editor reveal whether or not
+an expression is a given type.
```
[the remark this answers is not in the document]
Comment (david-a-wheeler) [reply docs/11/index.md:1078]: I think either way is fine, but since you prefer one form, let's use that one :-).

**Pull request merged**